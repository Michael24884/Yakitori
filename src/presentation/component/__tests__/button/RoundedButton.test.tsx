/**
 * @format
 */
import React from 'react';
import {RoundedButton} from '@components';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const instance = renderer.create(<RoundedButton />);
  expect(instance.toJSON()).toMatchSnapshot();
});
