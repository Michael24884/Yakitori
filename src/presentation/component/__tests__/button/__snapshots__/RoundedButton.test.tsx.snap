// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onLayout={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  pointerEvents="box-only"
  style={
    Object {
      "height": 44,
    }
  }
>
  <BVLinearGradient
    colors={
      Array [
        4293807771,
        4291848039,
      ]
    }
    endPoint={
      Object {
        "x": 0.5,
        "y": 1,
      }
    }
    locations={null}
    startPoint={
      Object {
        "x": 0.5,
        "y": 0,
      }
    }
    style={
      Object {
        "alignItems": "center",
        "borderRadius": 8,
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontWeight": "600",
        }
      }
    />
  </BVLinearGradient>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
          "bottom": 0,
          "left": 0,
          "overflow": "hidden",
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "borderRadius": 0,
        },
      ]
    }
  />
</View>
`;
